.bazelrc

```diff
@@ -64,6 +64,15 @@ build:clang-tsan --define tcmalloc=disabled
 # Needed due to https://github.com/libevent/libevent/issues/777
 build:clang-tsan --copt -DEVENT__DISABLE_DEBUG_MODE
 
+# Link (statically) against libc++.
+build:libc++ --action_env=CXXFLAGS=-stdlib=libc++
+build:libc++ --action_env=BAZEL_CXXOPTS=-stdlib=libc++
+build:libc++ --action_env=BAZEL_LINKLIBS=-l%:libc++.a:-l%:libc++abi.a
+build:libc++ --action_env=BAZEL_LINKOPTS=-lm:-static-libgcc
+build:libc++ --linkopt=-fuse-ld=lld
+build:libc++ --host_linkopt=-fuse-ld=lld
```

Comment:
this doesn't work on mac

Reply:
Yeah, I know by now... testing asan/tsan on macOS now (which takes forever).

Reply:
I pushed new version with Linux/macOS switch, PTAL.